scripts/spack/configs/ubuntu20.04/packages.yaml

```diff
@@ -4,13 +4,26 @@ packages:
     - spec: mpich@3.3.2
       prefix: /usr
     buildable: false
-
+  all:
+    variants: +pic
+    providers:
+      mpi: [mpich]
+      blas: [netlib-lapack]
+      lapack: [netlib-lapack]
+      pkgconfig: [pkg-config]
+      pkgconf: [pkg-config]
```

Comment:
Do we need both of these?

Reply:
I think so. I don't think pkgconfig is a virtual package so we have to be specific. I could be wrong though.